CHANGELOG

```diff
@@ -1,3 +1,9 @@
+2019-01-?? Release 0.9.?
```

Comment:
We should mention that `aptly::mirror::key` has changed behaviour and `aptly::mirror::keyserver` has been removed

Reply:
As this is a breaking change, let's bump to `1.0.0`